Userland/Applications/TerminalSettings/TerminalSettingsView.gml

```diff
@@ -0,0 +1,91 @@
+@GUI::Widget {
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10]
+        spacing: 5
+    }
+
+    @GUI::GroupBox {
+        title: "Background Opacity"
+        fixed_height: 100
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [16, 8, 8]
+            spacing: 16
+        }
+
+        @GUI::Label {
+            text: "Smaller values makes the terminal appear more transparent."
+            text_alignment: "TopLeft"
+        }
+
+        @GUI::OpacitySlider {
+            name: "background_opacity_slider"
+            min: 0
+            max: 255
+            orientation: "Horizontal"
+        }
+    }
+
+    @GUI::GroupBox {
+        title: "Terminal Font"
+        fixed_height: 100
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [16, 8, 8]
+            spacing: 16
+        }
+
+        @GUI::Label {
+            text: "By default, this is the system-wide fixed-width font."
```

Comment:
This is very opaque. How do I know if it's in "default font" state, or if I've specifically assigned a font that happens to be the default?
Instead, we could have a "Use system default" checkbox, that disables the font selector if checked.

Reply:
That's a great idea!